src/kemal/handler.cr

```diff
@@ -15,6 +15,7 @@ class Kemal::Handler < HTTP::Handler
 
   def add_route(method, path, &handler : Kemal::Context -> _)
     @routes << Route.new(method, path, &handler)
+    @routes << Route.new("HEAD", path, &handler) if method == "GET"
```

Comment:
This actually registers two routes if the request is a ```GET```. One for the normal ```GET```and 1 for ```HEAD```